src/display/api.js

```diff
@@ -783,14 +783,15 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
      * @param {number} scale The desired scale of the viewport.
      * @param {number} rotate Degrees to rotate the viewport. If omitted this
      * defaults to the page rotation.
+     * @param {boolean} dontFlip if true, axis Y will not be flipped.
```

Comment:
Apparently the existing JSDoc looks a bit weird here, but let's fix the *new* one added here:
```javascript
@param {boolean} dontFlip - (optional) If true, axis Y will not be flipped.
```

Reply:
ok

- [ ] update jsDOC